typescript/nomad-monitor/src/config.ts

```diff
@@ -90,17 +90,69 @@ function getNetworks() {
       break;
 
     case 'staging':
-      networks = ['kovan', 'moonbasealpha'];
+      networks = ['kovan', 'moonbasealpha', 'rinkeby'];
       break;
     
     default:
-      networks = ['kovan', 'moonbasealpha'];
+      networks = ['kovan', 'moonbasealpha', 'rinkeby'];
       break;
   }
 
   return networks;
 }
 
+function getReplicas(origin: string) {
```

Comment:
this is gonna be a PITA to maintain as we add more chains. 

i would suggest something like
```
const hub = HUB[environment];
const domains = DOMAINS[environment];
if (origin === hub) { 
   return domains.filter(d => d !== hub); 
} else { 
   return [hub]; 
}
```
then you only ever need to add new domains to DOMAINS.
could probably make HUB and DOMAINS cleaner as well, but this is a minimal patch

Reply:
would even be cleaner to have some better way than hard-coding HUB and DOMAINS somewhere, but better than hard-coding everything imo